app/src/main/java/com/github/se/studybuddies/data/Contact.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.se.studybuddies.data
+
+data class Contact(val id: String, val members: Pair<String, String>, val showOnMap: Boolean) {
+  companion object {
+    fun empty(): Contact {
+      return Contact(id = "", members = Pair("", ""), false)
+    }
+  }
+}
+
+class ContactList(private val contacts: List<Contact>) {
+  fun getAllTasks(): List<Contact> {

```

Comment:
The empty() method in the companion object of Contact might be redundant if not used.